azurerm/resource_arm_key_vault_secret.go

```diff
@@ -280,6 +334,14 @@ func resourceArmKeyVaultSecretRead(d *schema.ResourceData, meta interface{}) err
 	d.Set("version", respID.Version)
 	d.Set("content_type", resp.ContentType)
 
+	if v := resp.Attributes.NotBefore; v != nil {
```

Comment:
`Attributes` has the potential to be nil so we should that it isn't nil before referencing `NotBefore` and `Expires`

Reply:
good call, added a nil check for `Attributes`